lib/command/CommandStack.js

```diff
@@ -456,7 +456,7 @@ CommandStack.prototype._popAction = function() {
   actions.pop();
 
   if (!actions.length) {
-    this._eventBus.fire('elements.changed', { elements: unique(dirty) });
+    this._eventBus.fire('elements.changed', { elements: uniqueBy(function(e) { return e; }, dirty) });
```

Comment:
min-dash does not support comparing objects without a given key/property or transforming the object into a key (string or number). I am not sure what key/property would be best to determine if the objects in dirty are unique.
Currently it just checks if the objects are the same object, but with they are both different objects with the same values, there is no way for min-dash's unique to determine that.
I could stringify the object and which would allow for comparing the string, but that is performant and I believe different property orders will result in different strings even if all of the properties are the same between the objects.

Reply:
The elements `id` is the key you're searching for.